be/src/service/doris_main.cpp

```diff
@@ -432,6 +433,12 @@ int main(int argc, char** argv) {
     doris::ExecEnv::init(exec_env, paths);
     doris::TabletSchemaCache::create_global_schema_cache();
 
+    // init s3 write buffer pool
+    doris::io::S3FileBufferPool* s3_buffer_pool = doris::io::S3FileBufferPool::GetInstance();
+    s3_buffer_pool->init(doris::config::s3_write_buffer_whole_size,
+                         doris::config::s3_write_buffer_size,
+                         exec_env->buffered_reader_prefetch_thread_pool());
```

Comment:
Is it intent to reuse the "reader thread pool"? 
It seems not a good idea. I suggest making them isolated.

Reply:
I didn't change the origin logic.
Maybe we can use separate pool in next PR